### PR TITLE
delete user

### PR DESCRIPTION
**Pull request by tungblvt**
==> Title: delete user

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/13569

### CODE REVIEW

**Comment by tungblvt** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -18,9 +20,24 @@ def create
     end
   end
 
+  def destroy
+    @user = User.find_by id: params[:id]
```
Comment (huuphuc93) [inline app/controllers/users_controller.rb]: check đk k tìm thấy  @user

**Comment by tungblvt** [comment]
ready

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -18,9 +20,28 @@ def create
...
+    if @user
+      if @user.destroy
+        flash[:success] = t "list_users.delete_successfully"
+        redirect_to users_url
+      else
+        flash[:danger] = t "list_users.some_thing_was_wrong"
+      end
+    else
+    redirect_to static_pages_home_path
```
Comment (huuphuc93) [inline app/controllers/users_controller.rb]: indent

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -18,9 +20,28 @@ def create
...
+  end
+
   private
 
   def user_params
     params.require(:user).permit User::USER_PARAMS
   end
+
+  def is_admin
+    return if current_user.role == "admin"
```
Comment (huuphuc93) [inline app/controllers/users_controller.rb]: "admin"  vào Settings

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -23,8 +23,8 @@
           <td><%= user.role %></td>
           <td>
             <a href=""><%= t "list_users.Edit" %></a>
-            <a href=""><%= t "list_users.Delete" %></a>
-            <a href=""><%= t "list_users.Lock" %></a>
+            <%= link_to t("list_users.Delete"), user, method: :delete,
+                        data:{ confirm: t("list_users.are_you_sure")} %>
```
Comment (huuphuc93) [inline app/views/users/index.html.erb]: indent

**Comment by tungblvt** [comment]
ready

**Pull request merged**